lib/hdfsutils/utils/util.rb

```diff
@@ -36,5 +37,40 @@ def initialize(name, argv, optsproc = nil)
     def run
       fail 'Subclass of Util must override Util::run'
     end
+
+    #
+    # Useful operations common between utilities
+    #
+    def stat?(path)
+      stat = nil
+      begin
+        stat = @client.stat(path)
+      # rubocop:disable Lint/HandleExceptions
+      rescue WebHDFS::FileNotFoundError
+        # fall through, leave stat == nil
+      end
+      # rubocop:enable Lint/HandleExceptions
+      stat
+    end
+
+    def list?(path)
```

Comment:
list? is not a predicate (returns only true or false), so it should not end in a question mark.  Please rename list?.